examples/src/MenuStructure.tsx

```diff
@@ -38,6 +44,9 @@ export const mainMenu: StateSpace<UIFragment, UIFragmentSpec, {}> = [
             main: MemTableAnt,
         },
         tokenMapping: ['groupId', 'objectId'],
+        // This is where we attach this scheme to this part of the menu tree, meaning that we will
+        // be able to address these states without knowing the path to here.
+        navRef: memTableRef,
```

Comment:
A question: the navRef also implicitly defines the tokens, so you have to define them twice. Can this be streamlined more?